Blackstar/BlackstarClient.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Blackstar
+{
+    public class BlackstarClient
+    {
+        readonly HttpClient _client;
+        readonly BlackstarServerRoutes _routes;
+
+        public BlackstarClient(string serverUrl)
```

Comment:
Personally I've started accepting strings less often than `Uri` at the outer layer, since callers mostly end up having to construct one from a string at the call site anyway.